repose-aggregator/commons/commons-utilities/src/main/scala/org/openrepose/commons/utils/servlet/http/HttpServletResponseWrapper.scala

```diff
@@ -223,6 +225,10 @@ class HttpServletResponseWrapper(originalResponse: HttpServletResponse, headerMo
     addHeader(name, DateUtils.formatDate(new Date(timeSinceEpoch)))
 
   override def addHeader(name: String, value: String): Unit = {
```

Comment:
Maybe. Originally it was `add`'ing, then we included `append`'ing. If we also include set'ing, then we also have to include `replace`'ing.

I'm good with it and it is probably the right thing to do.